rocketpy/stochastic/stochastic_model.py

```diff
@@ -514,11 +514,20 @@ def format_attribute(attr, value):
                 )
             elif isinstance(value, tuple):
                 nominal_value, std_dev, dist_func = value
-                return (
-                    f"\t{attr.ljust(max_str_length)} "
-                    f"{nominal_value:.5f} ± "
-                    f"{std_dev:.5f} ({dist_func.__name__})"
-                )
+                if callable(dist_func) and dist_func.__name__ == "uniform":
+                    mean = (std_dev + nominal_value) / 2
+                    half_range = (std_dev - nominal_value) / 2
+                    return (
+                        f"\t{attr.ljust(max_str_length)} "
+                        f"{mean:.5f} ± "
+                        f"{half_range:.5f} ({dist_func.__name__})"
+                    )
```

Comment:
I think using `±` here makes it a bit confusing.

I suggest just putting a comma like:

```suggestion
                if callable(dist_func) and dist_func.__name__ == "uniform":
                    lower_bound = nominal_value
                    upper_bound = std_dev
                    return (
                        f"\t{attr.ljust(max_str_length)} "
                        f"{lower_bound:.5f}, {upper_bound:.5f} ({dist_func.__name__})"
                    )
```